src/app/components/tooltip/tooltip.ts

```diff
@@ -297,7 +297,13 @@ export class Tooltip implements AfterViewInit, OnDestroy {
 
     onMouseLeave(e: MouseEvent) {
         if (!this.isAutoHide()) {
-            const valid = DomHandler.hasClass(e.target, 'p-tooltip') || DomHandler.hasClass(e.target, 'p-tooltip-arrow') || DomHandler.hasClass(e.target, 'p-tooltip-text') || DomHandler.hasClass(e.relatedTarget, 'p-tooltip');
+            const valid =
+                DomHandler.hasClass(e.target, 'p-tooltip') ||
```

Comment:
Your code works perfectly thanks!
If you leave without `e.target` works good too and less lines :)

![image](https://github.com/primefaces/primeng/assets/19764334/276b142a-f9a1-4e84-827e-f7dc5c5ee268)

Reply:
thanks for the suggestion @SoyDiego. I have pushed the changes!

Reply:
Happy to help 🚀